test/psse.jl

```diff
@@ -173,4 +170,23 @@ end
         setlevel!(getlogger(PowerModels), "error")
     end
 
+    @testset "three-winding transformer" begin
+        data_pti = PowerModels.parse_file("../test/data/pti/three_winding_test.raw")
+        data_mp = PowerModels.parse_file("../test/data/matpower/three_winding_test_mp.m")  # Matpower Translation of PTI file
+        data_pw = PowerModels.parse_file("../test/data/matpower/three_winding_test_pw.m")  # PowerWorld Translation of PTI file
```

Comment:
What utility do you think the the Matpower translations of `three_winding_test.raw` give us?  I am inclined to omit these.

Reply:
I was doing internal testing before I got the powerworld translations and neglected to remove it.